src/encoded/static/components/app.js

```diff
@@ -1164,6 +1164,7 @@ class App extends React.Component {
                     <meta charSet="utf-8" />
                     <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
                     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+                    {this.props.context['@type'].includes('Gene') ? <meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests" /> : null}
```

Comment:
The reason for this line will be a mystery in a month, so I would add a comment on the line before. Since this part is JSX and not Javascript, you’ll need to do it in a weird way. On the line before, you’d need something like:

```
{/* Here is a JSX comment */}
```

Reply:
Thanks. Added 3 comments.